codegen/test_data/clients/bar.gogen

```diff
@@ -532,7 +532,7 @@ func (c *barClient) ArgWithParams(
 	req := zanzibar.NewClientHTTPRequest(c.clientID, "ArgWithParams", c.httpClient)
 
 	// Generate full URL.
-	fullURL := c.httpClient.BaseURL + "/bar" + "/" + string(r.UUID) + "/segment" + "/" + string(r.Params.UserUUID)
+	fullURL := c.httpClient.BaseURL + "/bar" + "/argWithParams" + "/" + string(r.UUID) + "/segment" + "/" + string(r.Params.UserUUID)
```

Comment:
Is `fmt.Sprintf()` faster than string concatenation?

Reply:
not sure.

Reply:
fmt.Sprintf allocates a new string object and it's slow, use https://golang.org/pkg/bytes/#Buffer.WriteString if performance is a concern, which seems not here